src/data/data.import.js

```diff
@@ -0,0 +1,14 @@
+import movies from './mov.json';
+
+const dataMovies = movies.map((film) => ({
+    idFilm: film.id,
+    titleFilm: film.title,
+    voteAverageFilm: film.vote_average,
+    releaseDateFilm: film.release_date,
+    posterPathFilm: film.poster_path,
+    genresFilm: film.genres,
+    runtimeFilm: film.runtime,
+    overviewfilm: film.overview,
```

Comment:
About Film and film in the end of fileds. Movies array is list of movie. Movie is entity. Inside entity we are locate in context entity and intend all filds as owned entity. And we can use naming without clarifing.
For example: movie.id, movie.title, movie,genres etc.

And use destructuring,